README.md

```diff
@@ -48,18 +89,25 @@ Precompiled actor bundles are provided as [release binaries][releases] in this r
 We usually release all actors, the runtime, and the state abstraction at the same time. That means releasing:
 
 - `fil_actors_runtime`
+- `fil_actors_evm_shared`
 - `fil_actor_account`
 - `fil_actor_cron`
+- `fil_actor_datacap`
+- `fil_actor_eam`
+- `fil_actor_ethaccount`
+- `fil_actor_evm`
 - `fil_actor_init`
 - `fil_actor_market`
 - `fil_actor_miner`
 - `fil_actor_multisig`
 - `fil_actor_paych`
+- `fil_actor_placeholder`
 - `fil_actor_power`
 - `fil_actor_reward`
 - `fil_actor_system`
 - `fil_actor_verifreg`
 - `fil_builtin_actors_state`
+- `vm_api`
```

Comment:
does this belong in the release list?

Reply:
It doesn't.

Also, this entire section is out of date.

1. We don't publish any crates now.
2. We release by bumping the version in the root `Cargo.toml`. That automatically applies to all the actors at the same time.

I'll push some fixes to your branch.

Reply:
Pushed, LGTY?